apps/web/src/client/components/file-uploader/_client-actions.js

```diff
@@ -31,6 +44,20 @@ const buildErrorListItem = (uploadedFile, message) => {
 				</li>`;
 };
 
+/**
+ *
+ * @param {{show: boolean}} options
+ * @param {Element} uploadForm
+ */
+export const buildProgressMessage = ({ show }, uploadForm) => {
```

Comment:
this is just for testing out the function. the real feature is yet to be defined